### DISCUSSION TITLE
update from Apache NetBeans 20 to Apache NetBeans 21

### DISCUSSION THREAD

**Discussion by GreyZeng** [apache/netbeans#7122] [Q&A] [answered]
the latest version is : Apache NetBeans 21
my version: Apache NetBeans 20

when I click 'Help --> Check for updates' on toolbars

it shows that the version is the latest, shouldn't it show that it needs to be updated to Apache NetBeans version 21?

What is the hot update alternative to manually downloading the installer to install Apache NetBeans version 21?
![微信截图_20240301143050](https://github.com/apache/netbeans/assets/12782103/fa8e2722-33b3-4d90-8f36-63102cfae407)
![微信截图_20240301143103](https://github.com/apache/netbeans/assets/12782103/24495a46-8d88-4afd-86cb-282d02e3c2b6)

**Comment by neilcsmith-net** [comment, marked as answer]
It will show you that NB21 is available now! :smile:  We normally leave it a week or so before pushing the notification to users of the previous release.  Bit later than normal this time - apologies and thanks for the hint!

There is no hot update to the latest release - that's not feasible.  All the NBPackage built installers - so all of the community installers with JDK, plus all the ASF installers except Windows - are designed to update in place.  At some point we'll hopefully move the ASF Windows installer to the same infrastructure - some changes needed in the Windows launcher to make that feasible.
